Tiltfile

```diff
@@ -17,28 +17,23 @@ custom_build(
     outputs_image_ref_to='result/ref',
 )
 
-# Load the latest CRDs from Nix
-watch_file('result')
-if os.path.exists('result'):
-   k8s_yaml('result/crds.yaml')
-
 # We need to set the correct image annotation on the operator Deployment to use e.g.
 # oci.stackable.tech/sandbox/opa-operator:7y19m3d8clwxlv34v5q2x4p7v536s00g instead of
 # oci.stackable.tech/sandbox/opa-operator:0.0.0-dev (which does not exist)
 k8s_kind('Deployment', image_json_path='{.spec.template.metadata.annotations.internal\\.stackable\\.tech/image}')
 k8s_kind('DaemonSet', image_json_path='{.spec.template.metadata.annotations.internal\\.stackable\\.tech/image}')
 
-# Exclude stale CRDs from Helm chart, and apply the rest
-helm_crds, helm_non_crds = filter_yaml(
-   helm(
-      'deploy/helm/' + operator_name,
-      name=operator_name,
-      namespace="stackable-operators",
-      set=[
-         'image.repository=' + registry + '/' + operator_name,
-      ],
-   ),
-   api_version = "^apiextensions\\.k8s\\.io/.*$",
-   kind = "^CustomResourceDefinition$",
-)
-k8s_yaml(helm_non_crds)
+# Optionally specify a custom Helm values file to be passed to the Helm deployment below.
+# This file can for example be used to set custom telemetry options (like log level) which is not
+# supported by helm(set).
+helm_values = settings.get('helm_values', None)
+
+k8s_yaml(helm(
+   'deploy/helm/' + operator_name,
+   name=operator_name,
+   namespace="stackable-operators",
+   set=[
+      'image.repository=' + registry + '/' + operator_name,
+   ],
+   values=helm_values
+))
```

Comment:
We might be able to reduce this PR now:

https://github.com/stackabletech/operator-templating/pull/554/files

Reply:
The templating PR should be merged first then, so that we can incorporate its changes into this PR.

Reply:
Merged in fe17308.